en-US/Revision_History.xml

```diff
@@ -8,7 +8,36 @@
 	 <simpara>
 		<revhistory>
 			<revision>
-				<revnumber>3-0</revnumber>
+				<revnumber>5-1</revnumber>
+				 <date>January 2022</date>
+				 <author>
+					<firstname>Julian</firstname>
+					 <surname>Cable</surname>
+					 <email>jcable@redhat.com</email>
+				</author>
+				 <revdescription>
+					<simplelist>
+						<member>Minor update to address some reported issues:</member>
+						<member>Accessing IBM Style online.</member>
+						<member>Updated guidance for long commands.</member>
+						<member>Updated guidance for referring to object names.</member>
+						<member>Updated guidance for non-breaking spaces.</member>
+						<member>All punctuation, including periods and commas, is now placed outside quotation marks unless the punctuation is part of the quotation.</member>
```

Comment:
How about something simple, such as "Updated guidance for punctuation with quotation marks"?